apps/cloud-player/README.md

```diff
@@ -0,0 +1,53 @@
+# Cloud Player
+
+A generic player which autonomously manges audio focuses and player resources.
+
+## URLs
+
+### yoda-app://cloud-player/play
+
+Queries:
+- `text`: String. Optional speech synthesis text.
+- `url`: String. Optional media url.
+- `transient`: Enum `0`, `1`. Optional flag indicates if the media was expected to be requested with transient focus. Defaults to `1`
+- `sequential`: Enum `0`, `1`.  Optional flag indicates if the speech synthesis and media should be played in sequential order but not simultaneously. Defaults to `0`.
+- `mediaId`: Optional media id tag. Cloud Player does nothing with it.
```

Comment:
Shall we decouple the cloud player with the specific id? It seems related with the specific speech platform like Google Assistant or Alexa?

Reply:
Cloud player does nothing with these id. It just tag the voice instance with the id for later inspection.

Reply:
Rename it as `tag`?